.github/workflows/agw-coverage.yml

```diff
@@ -110,22 +110,24 @@ jobs:
         run: |
           echo "Available storage:"
           df -h
-      - name: Notify Bazel C/C++ coverage failure to slack
+      # Slack notification configuration
+      - name: "Set Slack channel #ci as default"
+        run: echo "webhook=${{ secrets.SLACK_WEBHOOK }}" >> $GITHUB_ENV
+      - name: "Set Slack channel #ci-info for failures"
+        run: echo "webhook=${{ secrets.SLACK_WEBHOOK_CI }}" >> $GITHUB_ENV
+        if: failure()
+      - name: "Set Slack channel #ci-artifacts for successes"
+        run: echo "webhook=${{ secrets.SLACK_WEBHOOK_OSS }}" >> $GITHUB_ENV
+        if: success()
+      - name: Notify Slack of job status
         if: |
-          failure() &&
-          (steps.bazel-cc-codecoverage.conclusion == 'failure' ||
-          steps.c-cpp-codecov-upload.outcome == 'failure') &&
-          github.event_name == 'push'
-        uses: rtCamp/action-slack-notify@12e36fc18b0689399306c2e0b3e0f2978b7f1ee7 # pin@v2.2.0
-        env:
-          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_TITLE: "C / C++ code coverage with Bazel"
-          SLACK_USERNAME: ${{ github.workflow }}
-          SLACK_ICON_EMOJI: ":boom:"
-          SLACK_COLOR: "#FF0000"
-          SLACK_FOOTER: ' '
-          MSG_MINIMAL: actions url,commit
-
+          steps.bazel-cc-codecoverage.conclusion == 'failure' ||
```

Comment:
The condition here is not equivalent to the before state. On `pull_request` no Slack message should be sent. And the secret may not even be available.

In general I think we should not change the `if` conditions unless there is a reason. In this case the channel might be spammed with messages or else all PR runs will fail because they do not have access to the secret.

Reply:
On second thought, I completely disagree! We should drop those if statements completely. Those special bits and pieces with no real benefit are the culprits, that make the code a mess.

Anyway I find the comment somewhat dialectic. No credential access on pull request means no spamming can occur. Spamming from pull request, would mean, that you do no have issues with credentials. What shall it be now?

In any case, do not worry about the secrets, they are accounted for within the composite action.